src/plugins/ovp/services/KalturaEntrySubscriber.js

```diff
@@ -19,12 +20,13 @@ export default class KalturaEntrySubscriber extends EventSubscriber {
   onInitForm(formEvent) {
     const data = formEvent.getData();
     const node = formEvent.getMessage();
-    const syncedAt = node.get('kaltura_synced_at') ? Microtime.fromString(String(`${node.get('kaltura_synced_at')}000000`)) : null;
 
+    data.kalturaSyncedAt = node.get('kaltura_synced_at')
```

Comment:
@efjacobson - same as above:
```suggestion
    data.kalturaSyncedAt = node.has('kaltura_synced_at')
```